src/dc_chat.rs

```diff
@@ -776,14 +776,15 @@ unsafe fn get_parent_mime_headers(
         || parent_in_reply_to.is_null()
         || parent_references.is_null())
     {
+        // prefer a last message that isn't from us
         success = (*chat)
             .context
             .sql
             .query_row(
                 "SELECT rfc724_mid, mime_in_reply_to, mime_references \
-                 FROM msgs WHERE timestamp=(SELECT max(timestamp) \
+                 FROM msgs WHERE chat_id=? AND timestamp=(SELECT max(timestamp) \
                  FROM msgs WHERE chat_id=? AND from_id!=?);",
-                params![(*chat).id as i32, 1],
+                params![(*chat).id as i32, (*chat).id as i32, 1],
```

Comment:
While updating this statement, could you also use `DC_CONTACT_ID_SELF` from constants.rs for the from_id?

Reply:
good idea